lib/Math/Sequences/Integer.pm

```diff
@@ -419,7 +419,9 @@ our @A001065 is export = ℕ.map: -> $n {
     [+] (1..^$n).grep: -> $i {$n %% $i};
 }
 # A001057 / all integers
-our @A001057 is export = 1, &NOSEQ ... *;
+our @A001057 is export = (lazy gather for 𝕀 -> $n
+    { $n ?? take $n, -$n !! take 0 }
+).flat;
```

Comment:
Suggest more readable alternative: `flat lazy gather for 𝕀 -> $n { take $n ?? $n, -$n !! 0 }`